decidim-comments/app/cells/decidim/comments/comments_cell.rb

```diff
@@ -39,6 +74,30 @@ def user_comments_blocked_warning
 
       private
 
+      def supports_two_columns_layout?
+        model.respond_to?(:comments_layout) && model.two_columns_layout?
+      end
+
+      def render_comments_in_two_columns
+        if model.closed?
+          top_comment_in_favor = comments_in_favor.order(up_votes_count: :desc).first
+          top_comment_against = comments_against.order(up_votes_count: :desc).first
+
+          other_comments_in_favor = comments_in_favor.where.not(id: top_comment_in_favor&.id)
+          other_comments_against = comments_against.where.not(id: top_comment_against&.id)
```

Comment:
we also need to order by creation asc all the comments after the initial comment